ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -78,6 +78,61 @@ objects:
 
     containers:
 
+    - name: lease
+      image: ci/boskoscli:latest
+      terminationMessagePolicy: FallbackToLogsOnError
+      resources:
+        requests:
+          cpu: 10m
+          memory: 10Mi
+        limits:
+          memory: 200Mi
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-profile
+        mountPath: /tmp/cluster
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      env:
+      - name: CLUSTER_TYPE
+        value: ${CLUSTER_TYPE}
+      - name: CLUSTER_NAME
+        value: ${NAMESPACE}-${JOB_NAME_HASH}
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        trap 'kill $(jobs -p); exit 0' TERM
+
+        if [[ "${CLUSTER_TYPE:-}" != "aws" ]]; then
+          echo "[WARNING] No resource leasing is defined for \"${CLUSTER_TYPE}\" clusters"
+          exit 0
+        fi
+
+        resource="$( boskosctl --server-url http://boskos --owner-name "${CLUSTER_NAME}" acquire --type aws-quota-slice --state clean --target-state leased --timeout 30m )"
```

Comment:
Is http://boskos resolved to some CI-cluster service via Kubernetes routing magic?  I've only briefly skimmed [the boskos README][1], so I'm not familiar with the deployment.  Is `--timeout 30m` the length of the lease once acquired, or a limit on how long `boskosctl` can take to acquire a lease of unlimited length?

[1]: https://github.com/kubernetes/test-infra/tree/master/boskos

Reply:
It should be a cluster service if I did it right. That's the timeout on waiting for a lease to be available. I didn't write it to be unlimited (and the hard timeout on jobs makes that less than useful) but I guess we can get away with like 2h30m or so waiting for quota before the test doesn't have enough time to finish anyway before the hard timeout.